<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSwap</class>
 <widget class="QWidget" name="NewSwap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="control::MwcLabelNormal" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>55</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Selling</string>
   </property>
  </widget>
  <widget class="control::MwcLineEditNormal" name="mwcAmountEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>45</y>
     <width>76</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20000</width>
     <height>40</height>
    </size>
   </property>
   <property name="maxLength">
    <number>4096</number>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="label_2">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>55</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MWC for </string>
   </property>
  </widget>
  <widget class="control::MwcLineEditNormal" name="secondaryAmountEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>45</y>
     <width>76</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20000</width>
     <height>40</height>
    </size>
   </property>
   <property name="maxLength">
    <number>4096</number>
   </property>
  </widget>
  <widget class="control::MwcComboBox" name="secondaryCurrencyCombo">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>45</y>
     <width>96</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>55</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rate:</string>
   </property>
  </widget>
  <widget class="control::MwcLineEditNormal" name="exchangeRateEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>45</y>
     <width>116</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20000</width>
     <height>40</height>
    </size>
   </property>
   <property name="maxLength">
    <number>4096</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>230</y>
     <width>271</width>
     <height>206</height>
    </rect>
   </property>
   <property name="title">
    <string>Timeouts</string>
   </property>
   <widget class="control::MwcLineEditNormal" name="redeemTimeEdit">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>145</y>
      <width>226</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20000</width>
      <height>40</height>
     </size>
    </property>
    <property name="maxLength">
     <number>4096</number>
    </property>
   </widget>
   <widget class="control::MwcLabelSmall" name="label_5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>125</y>
      <width>226</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Coins redeem time (minutes)</string>
    </property>
   </widget>
   <widget class="control::MwcLabelSmall" name="label_4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>45</y>
      <width>226</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Message exchange time (minutes)</string>
    </property>
   </widget>
   <widget class="control::MwcLineEditNormal" name="messageExchageTimeEdit">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>65</y>
      <width>226</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20000</width>
      <height>40</height>
     </size>
    </property>
    <property name="maxLength">
     <number>4096</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>235</y>
     <width>341</width>
     <height>196</height>
    </rect>
   </property>
   <property name="title">
    <string>Confirmations</string>
   </property>
   <widget class="control::MwcLineEditNormal" name="mwcLockConfirmationsEdit">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>60</y>
      <width>196</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20000</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4096</number>
    </property>
   </widget>
   <widget class="control::MwcLabelSmall" name="label_6">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>40</y>
      <width>226</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Number of confirmatiions for MWC</string>
    </property>
   </widget>
   <widget class="control::MwcPushButtonNormal" name="calcMwcConfBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>86</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="toolTip">
     <string>Restore default settings for the wallet.</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="control::MwcLineEditNormal" name="secondaryLockConfirmationEdit">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>135</y>
      <width>196</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20000</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4096</number>
    </property>
   </widget>
   <widget class="control::MwcLabelSmall" name="label_7">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>115</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Number of confirmatiions for Secondary</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>445</y>
     <width>661</width>
     <height>126</height>
    </rect>
   </property>
   <property name="title">
    <string>Buyer address</string>
   </property>
   <widget class="control::MwcLineEditNormal" name="destinationEdit">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>55</y>
      <width>466</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="maxLength">
     <number>1024</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="mwcmqsRadio">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>45</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MWC MQS</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="torRadio">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tor</string>
    </property>
   </widget>
   <widget class="control::MwcLabelSmall" name="label_8">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>35</y>
      <width>226</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
  </widget>
  <widget class="control::MwcLineEditNormal" name="secondaryAddressEdit">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>125</y>
     <width>651</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>1024</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="control::MwcLabelSmall" name="secondaryAddressLabel">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>105</y>
     <width>226</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Redeem address</string>
   </property>
  </widget>
  <widget class="control::MwcComboBox" name="lockOrderCombo">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>185</y>
     <width>246</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Seller Lock first</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Buyer Lock first</string>
    </property>
   </item>
  </widget>
  <widget class="control::MwcLabelNormal" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Lock order:</string>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="createBtn">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>590</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Send MWC</string>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="backBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>595</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Send MWC</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>605</y>
     <width>142</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="control::MwcLabelProgress" name="progress">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control_desktop/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelSmall</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLineEditNormal</class>
   <extends>QLineEdit</extends>
   <header>control_desktop/MwcLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelProgress</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabelProgress.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelNormal</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcComboBox</class>
   <extends>QComboBox</extends>
   <header>control_desktop/MwcComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
